geometry/SettingsViewController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SettingsViewController.swift
+//  geometry
+//
+//  Created by Dmitry Tolmachev on 21.02.16.
+//  Copyright © 2016 Al Digit. All rights reserved.
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController
+{
+
+    @IBAction func doneUnwindAction(unwindSeque: UIStoryboardSegue) {
+        if let tViewController = unwindSeque.sourceViewController as? SettingsViewController {
```

Comment:
tViewController is too generic for a variable name. Please use more specific names. For instance "settingsViewController".